docs/04-community/02-rspack.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Rspack Integration
+nav_title: Rspack
+description: Using the next-rspack plugin to use Rspack as bundler
```

Comment:
Should next-rspack be inline code

Reply:
Maybe to avoid "using" twice
Using the next-rspack plugin to enable Rspack as bundler